jax/_src/pallas/mosaic/pallas_call_registration.py

```diff
@@ -96,7 +96,7 @@ def _lower_fun(*args):
     return mosaic.as_tpu_kernel(
         mosaic_module,
         out_avals,
-        backend=ctx.module_context.backend,
+        backend="tpu",
```

Comment:
Would it be incorrect to use `ctx.module_context.backend` here?

Reply:
I need to clean this `module_context.backend` in all lowering rules. Right now, it aborts of there are more than 1 lowering platforms, which would be the case here when we have multi-platform lowering.